app/models/deploy.rb

```diff
@@ -121,6 +121,34 @@ def self.expired
     joins(:job).where(jobs: { status: 'pending'} ).where("jobs.created_at < ?", threshold)
   end
 
+  def csv_buddy
+    if not (BuddyCheck.enabled? && stage.production?)
+      "Buddy Not Required"
+    elsif buddy.nil? && pending?
+      "Pending Buddy"
+    elsif buddy.nil?
+      "No Buddy"
+    elsif (user.id == buddy.id)
+      "Buddy Bypassed"
+    else
+      buddy.name
+    end
+  end
```

Comment:
Removed "buddy" from the column output, and also updated first condition to use the deploy_requires_approval? function instead of recreating the condition.